### ISSUE TITLE
DB queries

### ISSUE THREAD

**Issue by eshiota** [joaomdmoura/sweet#19]
It would be nice to be able to retrieve data from a DB when compiling the HTML. Coupling this feature with a template engine, it would be easy to iterate through a collection of data — i.e. printing a list of users.

**Comment by joaomdmoura** [comment]
Of Course, it seems pretty cool to me too, I certainly will implement it, probably in next version after 0.4.6 ;) I will let you know when it be done.

**Comment by joaomdmoura** [comment]
Srry i've commited it wrong, it's about issue #17

**Comment by joaomdmoura** [comment]
Okay, it will be implemented this feature is set up to the milestone v0.6.0
